CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 Change Log
 ==========
 
+[1.4.1 - 04-08-2018](https://github.com/bbaugher/apache_zookeeper/issues?milestone=20&state=closed)
+---------------------------------------------------------------------------------------------------
+
+  * [Bug] [Issue-53](https://github.com/bbaugher/apache_zookeeper/issues/53) : Update init.d/systemd to respect configuration from `config_dir`.
```

Comment:
Sorry I missed this PR somehow. If you remove this change (changelog is automatically generated) I will merge this

Reply:
Thanks @bbaugher. I've updated the PR.